helm-charts/common/llm-uservice/values.yaml

```diff
@@ -109,7 +112,6 @@ global:
   http_proxy: ""
   https_proxy: ""
   no_proxy: ""
-  HUGGINGFACEHUB_API_TOKEN: "insert-your-huggingface-token-here"
```

Comment:
API token not needed any more?

Reply:
Thanks for finding this. It's actually used implicitly by the underlying python module to access gated model. I'll add this back.